### PR TITLE
Created Runtime-testing doc and add other files

### PR DESCRIPTION
**Pull request by nandan75**

### CODE REVIEW

File: Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0
Comment (georgiesamaha) [inline Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0:1]: Does the dockerfile need to be uploaded to this repository as well as https://github.com/Sydney-Informatics-Hub/Rstudio-rnaseq-contained? Would be easiest to maintain in one place.

File: Scripts/Day2/Runtime-testing_D2.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  
+- I remember encountering a similar error when I had tried running a nfcore workflow on another Nimbus instance.  
+- I had previously asked this same question to Audrey and she had provided the following solution
+ 
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Audrey Stott commented:
+
+```
+Hi Nandan
```
Comment (georgiesamaha) [inline Scripts/Day2/Runtime-testing_D2.md]: Our repository is now public, not appropriate for us to reveal personal correspondence, can you reword this to something like: 


```
Pawsey helpdesk suggested issue was a [known issue with Apptainer/Singularity](https://github.com/apptainer/apptainer/issues/1258) and the following workaround: 

* Configure max_loop for the loop driver to a higher value. Setting it to the same as max loop devices in /usr/local/etc/singularity/singularity.conf, which is 256 by default.
* Must do this with a kernel CMDLINE entry, because loop is compiled into the kernel on Ubuntu. It is not a module.
* Add max_loop=256 to the GRUB_CMDLINE_LINUX value in /etc/default/grub
* Run update-grub2 as root.
* Reboot the system.
* Run cat /proc/cmdline and verify max_loop=256 is present.
```

File: Scripts/Day2/Runtime-testing_D2.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- This error was observed only a few times during testing - Running thr RSever command.  
+- ERROR: `“FATAL:   container creation failed: mount /proc/self/fd/3->/usr/local/var/singularity/mnt/session/rootfs error: while mounting image /proc/self/fd/3: failed to find loop device: could not attach image file to loop device: no loop devices available”`    
+
+I have used the following solution identified from blogs/ChatGPT and it has worked for a few of us at SIH:  
+- Check if the loop module is `loaded: lsmod | grep loop`  
+- If the loop module is not listed, you can load it using the following command: `sudo modprobe loop`  
+- Verify the loop devices availability: `sudo losetup -f`  
+  
+- I remember encountering a similar error when I had tried running a nfcore workflow on another Nimbus instance.  
+- I had previously asked this same question to Audrey and she had provided the following solution
```
Comment (georgiesamaha) [inline Scripts/Day2/Runtime-testing_D2.md]: Remove names please

File: Scripts/Day2/Runtime-testing_D2.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+ - The Rmarkdown was updated and excecuted.  
+ - The `knit` option was used from RStudio to create a output `html` file.  
+ - One known issue as noted below.
+
+
+### Intermittent problems when running the Rserver
+- This error was observed only a few times during testing - Running thr RSever command.  
+- ERROR: `“FATAL:   container creation failed: mount /proc/self/fd/3->/usr/local/var/singularity/mnt/session/rootfs error: while mounting image /proc/self/fd/3: failed to find loop device: could not attach image file to loop device: no loop devices available”`    
+
+I have used the following solution identified from blogs/ChatGPT and it has worked for a few of us at SIH:
```
Comment (georgiesamaha) [inline Scripts/Day2/Runtime-testing_D2.md]: Remove this line and replace with

```
Current workaround:
```

File: Scripts/Day2/Runtime-testing_D2.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+—
+
+Let us know how you go or if you need any help.
+
+Kind regards
+
+Audrey
+```
+
+Since the problem does not occur frequently (I have observed it only once on any of the instances I have worked so far). One of our collegues Giorgia has also encountered it on her insatnce. I have not tried the workaround as suggested by Audrey, yet.
```
Comment (georgiesamaha) [inline Scripts/Day2/Runtime-testing_D2.md]: Remove names.

File: Scripts/Day2/Runtime-testing_D2.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+Let us know how you go or if you need any help.
+
+Kind regards
+
+Audrey
+```
+
+Since the problem does not occur frequently (I have observed it only once on any of the instances I have worked so far). One of our collegues Giorgia has also encountered it on her insatnce. I have not tried the workaround as suggested by Audrey, yet.  
+Should we check with Audrey if this could be a possible issue on the trainee instances for the workshop and if we need to have any quick workarounds if this occures?
```
Comment (georgiesamaha) [inline Scripts/Day2/Runtime-testing_D2.md]: Remove name and replace with Pawsey helpdesk

**Review by georgiesamaha — CHANGES_REQUESTED** [review]
A few suggestions to remove personal correspondance and people's names from markdown documents. Keep in mind this repository is public.

File: Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0
Comment (nandan75) [reply Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0:1]: Not required, but I am using it in testing - So thought it is a good place in addition to the other repo. I was thinking of uploading the latest version of everything including all these essential files for testign to this repo. But we can figure out.

File: Scripts/Day2/Runtime-testing_D2.md (hunk 1 above)
Comment (nandan75) [reply Scripts/Day2/Runtime-testing_D2.md]: Sure

File: Scripts/Day2/Runtime-testing_D2.md (hunk 2 above)
Comment (nandan75) [reply Scripts/Day2/Runtime-testing_D2.md]: ok

File: Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0
Comment (nandan75) [reply Scripts/Day2/BioCommons_Dockerfile_4.1.0_V4.0:1]: For now I will marked this as resolved. I will remove the files if required at a later date

**Comment by nandan75** [comment]
Made the changes and pushed them to "Nandan" branch

**Review by georgiesamaha — APPROVED** [review]
